test/integration/model/find.test.js

```diff
@@ -1024,18 +1024,33 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         username: Sequelize.STRING
       }, { paranoid: true });
 
-      return User.sync({ force: true }).then(() => {
-        return User.bulkCreate([
-          {username: 'Bob'},
-          {username: 'Tobi'}
-        ]);
-      }).then(() => {
-        return User.destroy({ where: {username: 'Tobi'} });
-      }).delay(5000).then(() => {
-        return User.findAll({ paranoid: true });
-      }).then(users => {
-        expect(users.length).to.be.eql(1);
-      });
+
+      if (dialect === 'sqlite') 
```

Comment:
A timeout sounds very brittle, especially if you don't know why it appears and why it only appears only for SQLite. We have some flakiness in tests in general which are solved when restarting jobs, but if it happens consistently, it indicates some bug in the code...